independent-publisher-connectors/ORB Intelligence/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "apiKey_auth",
+          "description": "The apiKey_auth for this api",
+          "tooltip": "Provide your apiKey_auth",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",

```

Comment:
@aaryan2134 Please change it to `"#da3b01"` which is the default iconBrandColor for all the IP connectors.